go.mod

```diff
@@ -21,12 +21,12 @@ require (
 	github.com/spf13/cast v1.4.1
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.10.1
+	github.com/spf13/viper v1.11.0
 	github.com/stretchr/testify v1.7.1
 	github.com/tendermint/tendermint v0.34.16
 	github.com/tendermint/tm-db v0.6.7
-	google.golang.org/genproto v0.0.0-20211208223120-3a66f561d7aa
-	google.golang.org/grpc v1.44.0
+	google.golang.org/genproto v0.0.0-20220407144326-9054f6ed7bac
+	google.golang.org/grpc v1.45.0
```

Comment:
This is weird to me. I didn't know that viper had anything to do with genproto or grpc.

Reply:
Lots of extra changes in the go.sum too...

Reply:
I made this change myself. All I did was change viper to v1.11.0, then go mod tidy and ended up with the same set of changes. I did a `make run` and let it run through a few blocks just fine. I also did some light testing of the `provenanced config` commands and didn't see any problems.